Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# List of atoms to calculate
atoms = ['Li', 'Na']

# Store the results for each atom
results_dict = {}

# Loop over each atom and calculate orbitals and energies
for atom in atoms:
    results = calculate_orbitals_energies(atom)
    results_dict[atom] = results
    
    print(f"Results for {atom} atom:")
    print("Total Energy (Hartree):", results['total_energy'])
    print("Molecular Orbital Energies (Hartree):", results['mo_energies'])
    print("HOMO energy (Hartree):", results['homo_energy'])
    #print("Molecular Orbital Coefficients:")
    #print(results['mo_coefficients'])
    #print("\n" + "-"*50 + "\n")
    print("\n")
------------------

----- stdout -----
converged SCF energy = -7.31552598128109
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[3], line 9[0m
[1;32m      7[0m [38;5;66;03m# Loop over each atom and calculate orbitals and energies[39;00m
[1;32m      8[0m [38;5;28;01mfor[39;00m atom [38;5;129;01min[39;00m atoms:
[0;32m----> 9[0m     results [38;5;241m=[39m [43mcalculate_orbitals_energies[49m[43m([49m[43matom[49m[43m)[49m
[1;32m     10[0m     results_dict[atom] [38;5;241m=[39m results
[1;32m     12[0m     [38;5;28mprint[39m([38;5;124mf[39m[38;5;124m"[39m[38;5;124mResults for [39m[38;5;132;01m{[39;00matom[38;5;132;01m}[39;00m[38;5;124m atom:[39m[38;5;124m"[39m)

Cell [0;32mIn[2], line 42[0m, in [0;36mcalculate_orbitals_energies[0;34m(atom_symbol)[0m
[1;32m     40[0m [38;5;28;01mif[39;00m homo_index [38;5;241m>[39m[38;5;241m=[39m [38;5;241m0[39m:
[1;32m     41[0m     cube_filename [38;5;241m=[39m [38;5;124mf[39m[38;5;124m"[39m[38;5;132;01m{[39;00matom_symbol[38;5;132;01m}[39;00m[38;5;124m_HOMO.cube[39m[38;5;124m"[39m
[0;32m---> 42[0m     [43mcubegen[49m[38;5;241m.[39morbital(mol, cube_filename, mf[38;5;241m.[39mmo_coeff[:, homo_index])
[1;32m     43[0m     [38;5;28mprint[39m([38;5;124mf[39m[38;5;124m"[39m[38;5;124mHOMO cube file saved as: [39m[38;5;132;01m{[39;00mcube_filename[38;5;132;01m}[39;00m[38;5;124m"[39m)
[1;32m     45[0m     [38;5;66;03m# Visualize the cube file using py3Dmol[39;00m

[0;31mNameError[0m: name 'cubegen' is not defined

